jest.config.js

```diff
@@ -192,10 +192,7 @@ const config = {
   },
 
   // An array of regexp pattern strings that are matched against all source file paths, matched files will skip transformation
-  // transformIgnorePatterns: [
-  //   "\\\\node_modules\\\\",
-  //   "\\.pnp\\.[^\\\\]+$"
-  // ],
+  transformIgnorePatterns: ["/node_modules/(?!(react-leaflet|@react-leaflet).*)"],
```

Comment:
@mimischly7 one more comment here: There should only be one package name, which is either `react-leaflet` or `@react-leaflet`. Don't include both, just include the relevant one.

Reply:
I appears that the `react-leaflet...` package defines the public API whereas the a`@react-leaflet/core` package (installed automatically when installing the former) provides the concrete implementation of this API. Since both of these modules are in ES, I think we have to transpile both, let me know.

Reply:
(https://react-leaflet.js.org/docs/core-introduction/)

Reply:
@mimischly7 Ah that makes sense, thanks 👍